my-app/src/main/java/com/mycompany/app/findInverseSquareRoot.java

```diff
@@ -0,0 +1,26 @@
+package com.mycompany.app;
```

Comment:
use your editor to format this file. The spacing looks weird.

Reply:
Done